lib/gemstash/logging.rb

```diff
@@ -61,6 +63,7 @@ def initialize(app)
 
       def call(env)
         env["rack.logger"] = Gemstash::Logging.logger
+        env["rack.errors"] = Gemstash::Env.current.log_file
```

Comment:
I added this when I was getting errors down in rack when trying to use `rack.errors` to build a `rack.logger` object. I think that was actually having an exception when reporting an exception though.

I'll try reproducing to see how much this is needed